_includes/mermaid.html

```diff
@@ -0,0 +1,10 @@
+<script src="https://unpkg.com/mermaid@8.9.3/dist/mermaid.min.js"></script>
+<script>
+  $(document).ready(function () {
+    mermaid.initialize({
+      startOnLoad:true,
+      theme: "default",
+    });
+    window.mermaid.init(undefined, document.querySelectorAll('.language-mermaid'));
+  });
+</script>
```

Comment:
In my tests the initialization below did not work as expected for things like modifying the them. We are OK with just the mermaid JS for now.

```suggestion
<script src="https://unpkg.com/mermaid@10.7.0/dist/mermaid.min.js"></script>
```